addons/html_editor/static/src/core/history_plugin.js

```diff
@@ -448,14 +453,19 @@ export class HistoryPlugin extends Plugin {
 
     /**
      * @param { MutationRecord[] } records
+     * @param { boolean } [dispatch]
      */
-    handleNewRecords(records) {
+    handleNewRecords(records, dispatch = true) {
         const processedRecords = this.processNewRecords(records);
         if (processedRecords.length) {
             // TODO modify `handleMutations` of web_studio to handle
             // `undoOperation`
-            const stepState = this.stepsStates.get(this.currentStep.id);
-            this.getResource("handleNewRecords").forEach((cb) => cb(processedRecords, stepState));
+            if (dispatch) {
+                const stepType = this.currentStep.type;
+                this.getResource("handleNewRecords").forEach((cb) =>
+                    cb(processedRecords, stepType)
+                );
```

Comment:
This should be a `dispatchTo`, that was the whole point of calling the parameter `dispatch`... ^^'